include/zephyr/bluetooth/gatt.h

```diff
@@ -1618,6 +1618,18 @@ typedef uint8_t (*bt_gatt_notify_func_t)(struct bt_conn *conn,
 				      struct bt_gatt_subscribe_params *params,
 				      const void *data, uint16_t length);
 
+/** @typedef bt_gatt_subscribe_func_t
+ *  @brief Subscription callback function
+ *
+ *  If given, called with the subscription parameters given when subscribing
```

Comment:
It feels like this line should be for `bt_gatt_subscribe_params.subscribe`, and not the in the description of the typedef

Reply:
Agree

Reply:
Fixed